hack/go-build.sh

```diff
@@ -12,4 +12,3 @@ LDFLAGS="-s -w -X github.com/red-hat-storage/ocs-operator/controllers/defaults.I
 go build -tags 'netgo osusergo' -ldflags="${LDFLAGS}" -o ${OUTDIR_BIN}/ocs-operator ./main.go
 go build -tags 'netgo osusergo' -ldflags="${LDFLAGS}" -o ${OUTDIR_BIN}/metrics-exporter ./metrics/main.go
 go build -tags 'netgo osusergo' -ldflags="${LDFLAGS}" -o ${OUTDIR_BIN}/provider-api ./services/provider/main.go
-go build -tags 'netgo osusergo' -ldflags="${LDFLAGS}" -o ${OUTDIR_BIN}/status-reporter ./services/consumer/main.go
```

Comment:
we need to let build team know about this, so that they can also make this change in the DS.

Reply:
OK, looks like we are no longer building `status-reporter`, is that it?

Reply:
yes, It will be part of the new operator if I am not mistaken. @Madhu-1 can you keep me honest here?